tests/foreman/cli/test_report.py

```diff
@@ -82,11 +82,63 @@ def test_positive_install_configure_host(
     for client, puppet_proxy in zip(content_hosts, puppet_infra_host, strict=True):
         client.configure_puppet(proxy_hostname=puppet_proxy.hostname)
         report = session_puppet_enabled_sat.cli.ConfigReport.list(
-            {'search': f'host~{client.hostname},origin=Puppet'}
+            {'search': f'host~{client.hostname}'}
```

Comment:
Are we now relying completely on puppet for reporting, dont we have choice here for Ansible facts etc.